src/presentations/indexable-post-type-presentation.php

```diff
@@ -226,15 +226,11 @@ public function generate_open_graph_article_author() {
 	 * Generates the open graph article publisher.
 	 *
 	 * @return string The open graph article publisher.
+	 *
+	 * @codeCoverageIgnore Wrapper method.
 	 */
 	public function generate_open_graph_article_publisher() {
-		$open_graph_article_publisher = $this->context->open_graph_publisher;
-
-		if ( $open_graph_article_publisher ) {
-			return $open_graph_article_publisher;
-		}
-
-		return '';
+		return $this->context->open_graph_publisher;
```

Comment:
I would revert this change, because we want to be sure a string is returned.

Reply:
Reverted!